src/pipe_wrap.cc

```diff
@@ -188,6 +192,16 @@ void PipeWrap::SetPendingInstances(const FunctionCallbackInfo<Value>& args) {
 #endif
 
 
+void PipeWrap::Fchmod(const v8::FunctionCallbackInfo<v8::Value>& args) {
+  PipeWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap, args.Holder());
+  int mode = args[0]->Int32Value();
```

Comment:
Could you use the overload that takes a `context` argument, or (even better imo) write this as 
```c++
CHECK(args[0]->IsInt32());
int mode = args[0].As<Int32>()->Value();
```
?